### ISSUE TITLE
[bug] save_config modifies config in-place

### ISSUE THREAD

**Issue by valentingol** [valentingol/cliconfig#41]
Quick example:
```yaml
# default.yaml
param1: 2
param2@copy: param1
```
```python
# main.py
config = make_config("default.yaml")
show_config(config)
print()
save_config(config, "cfg.yaml")
show_config(config)
```
Returns:
```
param1: 2
param2: 2

param1: 2
param2@copy: param1
```
While the same thing is expected.

**Comment by valentingol** [comment]
Will be fixed soon.

**Comment by valentingol** [comment]
Fixed by [Pull Request #42](https://github.com/valentingol/cliconfig/pull/42)
